resources/home/dnanexus/utils/generate.py

```
@@ -0,0 +1,280 @@
+"""Functions for calling all the querying and generating the output data"""
+
+from collections import defaultdict
+import concurrent
+import os
+from typing import Tuple
```

Ruff (0.8.0)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)